samples/apps/copilot-chat-app/webapi/Controllers/SpeechTokenController.cs

```diff
@@ -29,16 +29,18 @@ public SpeechTokenController(IOptions<AzureSpeechOptions> options, ILogger<Speec
     }
 
     /// <summary>
-    /// Get an authorization token and region
+    /// Get an authorization token and region`
     /// </summary>
     [Route("speechToken")]
     [HttpGet]
     [ProducesResponseType(StatusCodes.Status200OK)]
     public async Task<ActionResult<SpeechTokenResponse>> GetAsync()
     {
-        if (string.IsNullOrWhiteSpace(this._options.Region))
+        // Azure Speech token support is optional. If the configuration is missing or incomplete, return a 404.
```

Comment:
> return a 404.
(nit), I think we return 200 with isSuccess=false in the response body. not a 404.

Reply:
woops! good catch.